.circleci/config.yml

```diff
@@ -68,6 +68,19 @@ commands:
             sudo apt-get install -y sshfs
             sudo sh -c 'echo user_allow_other >> /etc/fuse.conf'
 
+  "install-helm":
```

Comment:
So I too once added an "install-helm" to circle, but Luke pointed out since it might be something users run from their box, it'd be better to include installation of helm in the Makefile, which we have here:https://github.com/telepresenceio/telepresence/blob/release/v2/build-aux/tools.mk#L73-L85

So I'd suggest making helm a dependency of `make check` and then you can remove this part. Does that make sense / seem reasonable?

Reply:
Aah! Yes that makes a lot of sense 🙂